### PR TITLE
Fix all internal links

### PR DESCRIPTION
**Pull request by nfnt**
**Special notes for your reviewer**:
Needs another pass once https://github.com/kudobuilder/kudo.dev/pull/175 has been merged.

### CODE REVIEW

File: content/docs/developing-operators/templates.md [hunk 1]
```diff
@@ -16,31 +16,31 @@ Access instance parameters defined in `params.yaml`. E.g., to get the value of a
 :::
 
 ::: flag .OperatorName
-Gets the name of the operator (TODO: link to Operator)
+Gets the name of the [operator](../what-is-kudo.md#main-concepts)
```
Comment (zen-dog) [inline content/docs/developing-operators/templates.md:19]: I suggest to make the references absolute:
```suggestion
Gets the name of the [operator](/docs/what-is-kudo.md#main-concepts)
```

File: content/docs/developing-operators/templates.md [hunk 2]
```diff
@@ -16,31 +16,31 @@ Access instance parameters defined in `params.yaml`. E.g., to get the value of a
...
 :::
 
 ::: flag .Name
-Gets the name of the current instance (TODO: link to Instance)
+Gets the name of the current [instance](../what-is-kudo.md#main-concepts)
```
Comment (zen-dog) [inline content/docs/developing-operators/templates.md:23]:
```suggestion
Gets the name of the current [instance](/docs/what-is-kudo.md#main-concepts)
```

**Review by zen-dog — APPROVED** [review]
Lgtm, aside from relative links. Should we make **all** links absolute and starting with `/docs`?

File: content/docs/what-is-kudo.md [hunk 3]
```diff
@@ -108,7 +108,7 @@ By default, KUDO reserves three plan names for special purposes:
 
 - `deploy` (**required**) as mentioned above this plan deploys the application to the cluster. This is also the default plan which is used when a parameter value changes and does not have a _trigger_.
 - `upgrade` plan is used to upgrade an operator to a new version. If _upgrade_ is not defined, _deploy_ is used.
-- `cleanup` plan, when exists, is triggered when an operator is removed. This gives an operator a chance to have a graceful shutdown and cleanup resources. Read more about cleanup plans in our [documentaton](???).
+- `cleanup` plan, when exists, is triggered when an operator is removed. This gives an operator a chance to have a graceful shutdown and cleanup resources. Read more about cleanup plans in our [documentaton](developing-operators/plans.md#cleanup-plans).
```
Comment (porridge) [inline content/docs/what-is-kudo.md]:
```suggestion
- `cleanup` plan, when exists, is triggered when an operator is removed. This gives an operator a chance to have a graceful shutdown and cleanup resources. Read more about [cleanup plans](developing-operators/plans.md#cleanup-plans).
```

**Review by porridge — APPROVED** [review]
:-1: on changing links to absolute - IMHO this will just make restructuring content harder, and vuepress is already converting them to absolute links in the rendered site.
One nitpick inline - the linking words should match the linked content.

**Pull request merged**